File: dart/13b.dart
import 'dart:io';

void main(){
  print("YOU MUST INPUT T OR F");
  String userAns = stdin.readLineSync()!;
  while (userAns.toUpperCase() != "T") { // mix it up a bit
    print("YOU MUST INPUT T OR F");
    userAns = stdin.readLineSync()!;
  }
  print("Finally!");
}

File: c/13b.c
#include <stdio.h>
#include <string.h>

int main() {
  char input[10];

  do {
    printf("YOU MUST INPUT T OR F\n");
    scanf("%s", input);
  } while (strcmp(input, "t") != 0);

  printf("Finally!");

  return 0;
}
